### ISSUE TITLE
Error in console on dev server

### ISSUE THREAD

**Issue by keslerm** [officert/vue-slideout-panel#27]
I am receiving the following errors while in dev console

```
[Vue warn]: $attrs is readonly.

found in

---> <SlideoutPanel>
       <IndexView> at src/views/IndexView.vue
         <App> at src/App.vue
           <Root> vue2-slideout-panel.js:696
warn
vue2-slideout-panel.js:696
initRender/<
vue2-slideout-panel.js:4606
reactiveSetter
vue2-slideout-panel.js:1111
updateChildComponent
vue.esm.js:2838
prepatch
vue.esm.js:4144
patchVnode
vue.esm.js:5925
updateChildren
vue.esm.js:5822
patchVnode
vue.esm.js:5936:27
patch
vue.esm.js:6096
lifecycleMixin/Vue.prototype._update
vue.esm.js:2670
updateComponent
vue.esm.js:2788
get
vue.esm.js:3142
run
vue.esm.js:3219
flushSchedulerQueue
vue.esm.js:2981
nextTick/<
vue.esm.js:1837
flushCallbacks
vue.esm.js:1758



[Vue warn]: $listeners is readonly.

found in

---> <SlideoutPanel>
       <IndexView> at src/views/IndexView.vue
         <App> at src/App.vue
           <Root> vue2-slideout-panel.js:696
warn
vue2-slideout-panel.js:696
initRender/<
vue2-slideout-panel.js:4609
reactiveSetter
vue2-slideout-panel.js:1111
updateChildComponent
vue.esm.js:2839
prepatch
vue.esm.js:4144
patchVnode
vue.esm.js:5925
updateChildren
vue.esm.js:5822
patchVnode
vue.esm.js:5936:27
patch
vue.esm.js:6096
lifecycleMixin/Vue.prototype._update
vue.esm.js:2670
updateComponent
vue.esm.js:2788
get
vue.esm.js:3142
run
vue.esm.js:3219
flushSchedulerQueue
vue.esm.js:2981
nextTick/<
vue.esm.js:1837
flushCallbacks
vue.esm.js:1758
```

**Comment by ChadTaljaardt** [comment]
I am getting this issue as well

**Comment by keslerm** [comment]
If it helps here is the project/branch i'm using it on https://gitlab.com/aegiszero/laneway/tree/slideout/frontend

**Comment by officert** [comment]
@keslerm can you point me to where in the project you're using it?

**Comment by officert** [comment]
@keslerm @ChadTaljaardt hey guys, sorry for the slow response here. I just updates and released V2.0.0.

This fixes a bunch of problems around getting the component installed. Hoping it will fix the issue you're seeing.

https://github.com/officert/vue-slideout-panel/releases/tag/2.0.0

Though I haven't been able to reproduce, so I'll leave this one open for now.

**Comment by officert** [comment]
closing for now. feel free to re-open or submit another issue if you're still having issues.
